lib/helpers/files.js

```diff
@@ -70,11 +70,12 @@ function getMainJsPath(cwd) {
 		});
 }
 
-function getModuleName(cwd) {
+function getComponentName(cwd) {
 	return getPackageJson(cwd)
 		.then(pkgJson => {
 			if (pkgJson) {
-				return pkgJson.name;
+				const packageName = pkgJson.name;
+				return packageName ? packageName.split('/').pop() : packageName;
```

Comment:
do we require components be namespaced?

Reply:
no, most are but some aren't, the tests cover other cases :)

Reply:
`'o-thing'.split('/').pop() === 'o-thing';`
`'acme/o-thing'.split('/').pop() === 'o-thing';`

Reply:
I guess this means we continue to require component names to be unique, even if they are published to different places